locksmith/package.json

```diff
@@ -28,6 +29,12 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@types/cors": "^2.8.4",
+    "@types/es6-promise": "^3.3.0",
+    "@types/express": "^4.16.1",
+    "@types/jest": "^24.0.11",
+    "@types/nock": "^9.3.1",
+    "@types/supertest": "^2.0.7",
```

Comment:
Added some types definitions for third-party libs

Reply:
Are these used after the code has been transpiled? Should we move these to the development dependencies?

Reply:
There are a handful of cases where it's beneficial to have the type definitions in the regular dependencies: https://stackoverflow.com/questions/45176661/how-do-i-decide-whether-types-goes-into-dependencies-or-devdependencies

I think we should be safe keeping them as dev dependencies.

Reply:
so, dev dependencies will not be installed by CI... which means that if they are required to run tests, then the tests will fail.